pkgs/by-name/di/disnix-web-service/package.nix

```diff
@@ -38,8 +47,8 @@ stdenv.mkDerivation (finalAttrs: {
     xmlstarlet ed -L -a "//jar|//war" -t attr -n "modificationtime" -v "0" build.xml
 
     sed -i -e "s|#JAVA_HOME=|JAVA_HOME=${jdk}|" \
-       -e "s|#AXIS2_LIB=|AXIS2_LIB=${axis2}/lib|" \
-        scripts/disnix-soap-client
+      -e "s|#AXIS2_LIB=|AXIS2_LIB=${axis2}/lib|" \
+      scripts/disnix-soap-client
```

Comment:
nit: this might be a little more readable:
```bash
sed -i \
  -e "s|#JAVA_HOME=|JAVA_HOME=${jdk}|" \
  -e "s|#AXIS2_LIB=|AXIS2_LIB=${axis2}/lib|" \
  scripts/disnix-soap-client
```

Reply:
even better substituteInPlace